### PR TITLE
[sentiment] Add tfjs-node training script; Fix padding issue

### PR DESCRIPTION
**Pull request by caisq**
- train.js contains the training logic
- train.js uses data.js to load and format the training data
- sequence_utils.js is created to house the truncation/padding logic
  for both the tfjs-node training and in-browser inference
- README.md is updated

### CODE REVIEW

**Comment by caisq** [comment]
Gentle ping.

**Comment by nsthorat** [comment]
For the future for the one-liner with a return: 

`localZipFile.on('error', err => reject(err));`

**Comment by caisq** [comment]
@nsthorat Of course... Sorry I was confused for a moment. See https://github.com/tensorflow/tfjs-examples/pull/214
